### PR TITLE
Week 006

### PR DESCRIPTION
**Pull request by sendilkumarn**
r? @fitzgen

### CODE REVIEW

File: _posts/2018-08-13-this-week-in-rust-wasm-006.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  * WIP spec work: [here](https://yurydelendik.github.io/webassembly-dwarf/) and [here](https://gist.github.com/yurydelendik/802f36983d50cedb05f984d784dc5159)
+  * WIP branches: [here](https://github.com/yurydelendik/debugger.html/tree/x-scopes) and [here](https://github.com/yurydelendik/emscripten/tree/x-scopes)
+
+## News and Releases
+
+* [An official subcharter of the W3C wasm WG for debugging has been created](https://github.com/WebAssembly/debugging)
+
+## Articles, Blog Posts, and Talks
+
+* Cleaned up, curated and opinionated "Tools" section into "Tools You Should Know" [section](https://rustwasm.github.io/book/tools.html) to the book
```
Comment (mgattozzi) [inline _posts/2018-08-13-this-week-in-rust-wasm-006.md]: s/and opinionated/and added an opinionated
s/into/into the
s/to the book/as part of the book

File: _posts/2018-08-13-this-week-in-rust-wasm-006.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* Added a curated and opinionated "Crates You Should Know" [section](https://rustwasm.github.io/book/crates.html) to the book
+* [@ashleygwilliams](https://github.com/ashleygwilliams) super awesome talk at Rust Berlin - [slides](https://rustwasm.github.io/hello-wasm-bindgen/#1)
```
Comment (mgattozzi) [inline _posts/2018-08-13-this-week-in-rust-wasm-006.md]: s/super awesome/gave a super awesome

File: _posts/2018-08-13-this-week-in-rust-wasm-006.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+* [Saving millions of dollars with Rust and WebAssembly](https://twitter.com/jxxf/status/1027358517462626304)
+* AR paint project, built with js + rust/wasm at [wafflejs](https://wafflejs.com/) - [slides](https://slides.cwervo.com/wafflejs-webar-2018-08/#/)
```
Comment (mgattozzi) [inline _posts/2018-08-13-this-week-in-rust-wasm-006.md]: s/js/JS
s/rust/Rust

**Comment by sendilkumarn** [comment]
Lets merge this ❤️

**Comment by fitzgen** [comment]
I'll post to various social medias

**Pull request merged**